crates/solidity/outputs/cargo/build/src/main.rs

```diff
@@ -17,7 +17,8 @@ use solidity_language::GrammarConstructorDslV1;
 fn main() -> Result<()> {
```

Comment:
> The translation/construction is done at build “run-time” rather than inside the newly introduced definition “compiler” proc macro. There’s not much explicit reason for that, other than the fact that it was quicker for me to work on (quicker dev cycle) and the logic is just plain Rust code. This can be moved inside the proc macro later on, if we decide that it’s better that way.

TOL, what would be the use case for writing code inside the proc macro? This PR matches its intended usage, by calling `solidity_language::definition::SolidityDefinition::create()` and using its output to perform codegen. I wonder if I'm missing something.

Reply:
> will be able to just call (...)

This is more or less already happening in this PR without removing the v0/v1 facilities yet, right?

> what would be the use case for writing code inside the proc macro?

One use-case I can think of is less reliance on the custom build script infrastructure - grammar construction from the definition could be done at build-time already (maybe even report some errors?), since the data is static and already given to the macro (we actually did that in DSL v1, although using macros 1.0). Multiple consumers (parser codegen? ebnf generator, maybe something else) would consume the pre-generated `Grammar` once, without having to call `Grammar::from...()` which has to process the definition when called.

In general, I'd rather start simple (this PR) and maybe think if we can do better in the future, however it'd be great to make the build pipeline slightly less custom/more orthodox :smiley: